package.json

```diff
@@ -21,28 +21,30 @@
   },
   "dependencies": {},
   "devDependencies": {
-    "@alcalzone/release-script": "^3.7.0",
+    "@alcalzone/release-script": "^3.8.0",
     "@alcalzone/release-script-plugin-license": "^3.7.0",
     "@types/iobroker": "^5.0.6",
-    "eslint": "^8.55.0",
+    "chai": "^4.5.0",
+    "eslint": "^8.57.0",
     "eslint-config-standard-with-typescript": "^42.0.0",
-    "eslint-plugin-import": "^2.29.0",
-    "eslint-plugin-n": "^16.4.0",
-    "eslint-plugin-promise": "^6.1.1",
-    "typescript": "^5.3.3",
+    "eslint-plugin-import": "^2.29.1",
+    "eslint-plugin-n": "^16.6.2",
+    "eslint-plugin-promise": "^6.6.0",
     "gulp": "^4.0.2",
-    "mocha": "^10.2.0",
-    "chai": "^4.3.10"
+    "mocha": "^10.7.0",
+    "prettier": "^3.3.3",
+    "prettier-plugin-organize-imports": "^4.0.0",
+    "typescript": "~5.5.4"
   },
   "bugs": {
     "url": "https://github.com/ioBroker/ioBroker.type-detector/issues"
   },
   "main": "index.js",
   "files": [
```

Comment:
Puh for me it is a bit strange, that this `build.js` script copies the package.json into `build` and publish then happens from there. During cleanup I would try to achieve publish from root dir then just publish `build` dir and `LICENSE`. What do you think?

Reply:
As discussed offline we keep that way for now because it removed devDeps